src/main/resources/templates/login.html

```diff
@@ -1,68 +1,68 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
 
-<head th:replace="~{fragments/head-fragments :: head('User profile page')}"><title></title></head>
+<head th:replace="~{fragments/head-fragments :: head('Login')}"><title></title></head>
 
 <body>
 <section class="container-login-page">
 
-    <div class="container-login">
+    <!--/*@thymesVar id="registerStatusMsg" type=""*/-->
+    <h1 th:if="${registerStatusMsg != null}"
+        th:text="${registerStatusMsg}"
+        class="position-absolute top-0 start-0 text-success"
+    >
+    </h1>
+    <form th:action="@{/auth/login}"
+          method="post"
+          th:method="post"
+    >
+        <h3 class="text-white text-center">Please login</h3>
 
-        <h1 th:if="${registerStatusMsg != null}"
-            th:text="${registerStatusMsg}"
-            style="position: absolute; bottom: 50px; left: 50%; transform: translateX(-50%); color: darkgreen"
-        >
-        </h1>
-        <form th:action="@{/auth/login}"
-              method="post"
-              th:method="post"
-        >
-            <h2 class="text-white text-center">Welcome back</h2>
-            <!-- Email input -->
-            <div class="form-outline mb-4">
-                <input type="text"
-                       id="usernameOrEmail"
-                       th:name="usernameOrEmail"
-                       class="form-control"/>
-                <label class="form-label" for="usernameOrEmail">Username or email address</label>
-            </div>
-
-            <!-- Password input -->
-            <div class="form-outline mb-4">
-                <input type="password"
-                       id="password"
-                       th:name="password"
-                       class="form-control"/>
-                <label class="form-label" for="password">Password</label>
-            </div>
+        <hr>
+        <!-- Email input -->
+        <div class="form-floating ">
+            <input type="text"
+                   id="usernameOrEmail"
+                   th:name="usernameOrEmail"
+                   placeholder="Username or email address"
+                   class="form-control"/>
+            <label class="form-label" for="usernameOrEmail">Username or email address</label>
+        </div>
 
-            <!-- 2 column grid layout for inline styling -->
-            <div class="row mb-4">
-                <div class="col d-flex justify-content-center">
-                    <!-- Checkbox -->
-                    <div class="form-check">
-                        <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked/>
-                        <label class="form-check-label" for="form2Example31"> Remember me </label>
-                    </div>
-                </div>
+        <!-- Password input -->
+        <div class="form-floating ">
```

Comment:
"form-floating text-muted"

Reply:
the background-color of the form is background: rgb(72, 72, 72) which is a dark grey.
text-muted is a bootstrap class which provides a grey font color.

grey font color over grey background-color is not a best practice and we keep the font color of the inputs as #FFFFFF (white)